net-misc/asterisk-g729/asterisk-g729-11.0.3.1.9.ebuild

```diff
@@ -1,17 +1,15 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="6"
+EAPI="7"
```

Comment:
EAPI change requires a revbump every time :) for example, some default configure options might've changed.

Reply:
Policy?

As per note in the commit log, in spite of what you say there can be no possible changes to on disk.  In this case there is no ./configure, no make, no cmake of any kind.

If this is however policy, I'll action the rev-bump and re-push, just don't think there is a technical requirement (in this case, I do get that in the general case what you say is perfectly valid).

Reply:
"When I grew up" I used to follow this rule, 
https://wiki.gentoo.org/index.php?title=Proxied_Maintainer_FAQ&oldid=744352#When_an_ebuild_needs_a_revbump_.28-r1.29

but it's been changed to "check devmanual" which doesn't say anything about the subject. 
https://wiki.gentoo.org/wiki/Proxied_Maintainer_FAQ#When_an_ebuild_needs_a_revbump_.28-r1.29
https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html

This has been debated inside QA team and between developers with no common consensus being found. I'd still suggest to revbump on EAPI changes.

But I took a closer look, why do you even need to bump the EAPI for that ebuild? 6 is still perfectly valid and not going away in a long time. I now believe you shouldn't touch the ebuild at all :P

Reply:
There

Reply:
11.0.3.1.9 and 13.0.3.1.9 have matching KEYWORDS right? If you push these two ebuilds at the same, will anyone be installing 11.0.3.1.9? I don't see benefit in touching it, because it then kinda resets the stabilization clock too. If **something** depends on 11.0, I'd leave it as-is and return to update it later, after 13.0 has been on tree for a while. I'd also revbump, because potentially then the current version of 11.0 is proven to be working for everyone, if non-revbumped 11.0 breaks something.

So you can bump to 13.0., then remove 11.0 in few days if 13.0 seems to be working for everyone (and nothing explicitly depends on 11.0). Or you can call for stabilization immediately on 11.0 if that's what you want to do in the long run. Either way, I don't see benefit in **touching** the ebuild at all. I'm not having a bad day here, in the long run I want to avoid you and me spending time on useless stuff :P

Reply:
Right.

11.0.* goes against asterisk 11 which I'm fairly certain (sincerely hoping) I'm the only user left off.  Looking to make a push to get rid of that.  So hopefully by Feb/March we can call for removal of asterisk-11* and asterisk-g729-11.0* from the tree completely.

13.0.* goes againts asterisk 13.

So in the long run we need an ebuild for asterisk-g729 for each version of asterisk that we have in the tree.